builtin/v13/market/market_types.go

```diff
@@ -138,3 +139,31 @@ type GetDealActivationReturn struct {
 type GetDealSectorParams = DealQueryParams
 
 type GetDealSectorReturn = abi.SectorNumber
+
+type SettleDealPaymentsParams struct {
+	Deals bitfield.BitField
```

Comment:
Should be:

```go
type SettleDealPaymentParams = bitfield.BitField
```

(the upstream type "transparently" encodes as a bitfield without wrapping it in a struct)